docs/sdk/serverless-signing/conditional-signing.md

```diff
@@ -5,123 +5,56 @@ import TabItem from '@theme/TabItem';
 
 # Conditional Signing
 
-## Prerequisites
-
-- Familiarity with JavaScript
-- Basic understanding of [serverless signing](../serverless-signing/quick-start.md)
-
 ## Overview
-Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. This means that you can easily check any on-chain condition inside a Lit Action, which can be useful for generating proofs. This system can be used to uphold the integrity of data on the open web, in its function as a decentralized notary.
+Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. This means that you can easily check any on-chain condition inside a Lit Action, which can be useful for generating proofs.
```

Comment:
> Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. This means that you can easily check any on-chain condition inside a Lit Action, which can be useful for generating proofs.

Consider "Lit Actions inherit the powerful condition checking ability that Lit Protocol utilizes for [access control](https://developer.litprotocol.com/sdk/access-control/intro). You can easily check on or off-chain conditions inside of Lit Actions to generate proofs and condition-based transaction automations"